handlers/main.yml

```diff
@@ -1,5 +1,13 @@
 ---
 
 - name: restart newrelic-infra
-  service: name=newrelic-infra state=restarted
+  service:
+    name: newrelic-infra
+    state: restarted
+  when: nrinfragent_service_state != "stopped"
```

Comment:
Don't we need anything like `when: os != "windows"`? I assume that the `service` statement excludes any windows service.

Reply:
I don't believe so. The handlers are only called by the tasks and I put a filter there to call the handler specific for that OS. 

Examples:

```
- name: setup agent config
  template:
    src: newrelic-infra.yml.j2
    dest: /etc/newrelic-infra.yml
    mode: 0600
  notify: restart newrelic-infra
  when:
    - nrinfragent_state != "absent"
    - nrinfragent_os_name|lower != 'windows'

- name: setup agent config
  win_template:
    src: newrelic-infra.yml.j2
    dest: '%ProgramFiles%\New Relic\newrelic-infra\newrelic-infra.yml'
  notify: restart newrelic-infra (windows)
  when:
    - nrinfragent_state != "absent"
    - nrinfragent_os_name|lower == 'windows'
```